### DISCUSSION TITLE
It doesn't seem like modpack's `user` direcotry is working

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [gorilla-devs/ferium#114] [Q&A] [answered]
I currently want to play Sky Factory 4, which has two mods that denied 3rd party access, so I downloaded them manually, after that, I'm create the `user` folder myself, and do `ferium modpack upgrade` but ferium didn't copy them! Is there something I did wrong?

**Comment by theRookieCoder** [comment, marked as answer]
Modpacks currently do not have a `user` directory
